src/ServiceControl.Audit/Infrastructure/RavenDB/RavenHostBuilderExtensions.cs

```diff
@@ -17,6 +19,8 @@ static class RavenHostBuilderExtensions
 
                 serviceCollection.AddSingleton<IDocumentStore>(embeddedDocumentStore);
                 serviceCollection.AddHostedService<EmbeddedRavenDbHostedService>();
+
+                serviceCollection.AddSingleton<IBodyStorage, RavenAttachmentsBodyStorage>();
```

Comment:
Technically the previous code allowed customization of the body storage by dropping a dll into the bin folder that contains a feature that registers a body storage. But that would depend on the feature activation order so I think it is fine to lock this faulty extension point up (for now).

👍

Reply:
yeah it removes a potential "magic" extension point but if we start to make use of that, I think we should make it explicit (especially if there might be a chance that we won't use NSB internally anymore)